privacyidea/lib/config.py

```diff
@@ -861,3 +861,27 @@ def return_saml_attributes_on_fail():
     r = get_from_config(key="ReturnSamlAttributesOnFail", default=False,
                         return_bool=True)
     return r
+
+
+def get_privacyidea_node():
+    """
+    This returns the node name of the privacyIDEA node as found in the pi.cfg
+    file in PI_NODE.
+    If it does not exist, the PI_AUDIT_SERVERNAME is used.
+    :return:
+    """
+    node_name = current_app.config.get("PI_NODE",
+                                       current_app.config.get("PI_AUDIT_SERVERNAME", ""))
```

Comment:
Should we use a default value if neither PI_NODE nor PI_AUDIT_SERVERNAME are given? Right now, the function would return ``""``. What about something like ``"localnode"``?

Reply:
We should choose a very nasty node name, so that users are encourage to change it!

Reply:
"nnn" = Nasty Node Name ;-)

...ok, I go with "localnode".